### ISSUE TITLE
False positives in `declaration-property-value-no-unknown` rule when using arithmetic operations.

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint/stylelint#7947]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  margin-bottom: 0 - 8px;
  margin-left: 1px * 8px;
  margin-top: (8px + 1px);
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard"
  ],
  "rules": {
    "declaration-property-value-no-unknown": true
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLgWyQCcBzAS0wFoAjCHHCPeFABhQpQA4AHADwG4suAiXIUANjABmOJgEZeKAFSdeA7PiJlK9LkwAU3HigDUKeTwCUagL6DMIADQhJpCQDkkeOIhg9PXCQA6MABnEMdwCEwXYgQQDEwUFHQQXxwYTCgQlKYAbSwkpJSQnABPCTFyHApIaNJiChKkTKIoFIKUAF0HDpTCAFcJbIQ0DqKQWDAxIiQcUiiKLkIILhhCMooANyQxfpgKTAgKfswAa0OAd2wRnAGYDttMawjamIAxCEICHDiAKxCohFYFxwogEuMSuUYJVMDgcskQNN0iUUj1EgjIRUqjUojFGjhmlBWvCUkiYCiQFhntYgA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

No positives.

### What actually happened?

False positives are raised for properties that use values containing operations (e.g., arithmetic operations). The rule incorrectly flags these operations as unknown values.

**Comment by Mouvedia** [comment]
Maybe I am not up to date with the latest specifications but it seems to me that these value should be passed to `calc()`.
Are you using a preprocessor?
